src/components/articles/ArticleInlineList.jsx

```
@@ -0,0 +1,84 @@
+ */
+function ArticleInlineListItem({ itemWrapper }) {
+    return (
+        <li className={`article-inline-list-item text-4`}>
+            <Link href={itemWrapper.link?.href || null}
+                  tooltip={itemWrapper.link?.tooltip}
+                  metadata={itemWrapper.link?.metadata}>
+                <i className={`article-inline-list-item-icon ${itemWrapper.faIconWithFallback}`}
+                   style={itemWrapper.faIconStyle}/>
+
+                <span className={`article-inline-list-item-label`}
+                      dangerouslySetInnerHTML={{__html: itemWrapper.locales.label || itemWrapper.label || itemWrapper.placeholder}}/>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)